src/Nancy.Siren.Demo/Infrastructure/OrderItemViewModelWriter.cs

```diff
@@ -33,7 +33,7 @@ public Siren Write(IEnumerable<OrderItemViewModel> data, Uri uri)
 
            
 
-            sirenDoc.links = new List<Link> { new Link { href = uri + "/", rel = new[] { "self" } }, new Link{href = uri.ToString().Replace("items",""), rel = new []{"order"}} };
+            sirenDoc.links = new List<Link> { new Link { href = uri.ToString() , rel = new[] { "self" } }, new Link{href = uri.ToString().Replace("/items",""), rel = new []{"order"}} };  

```

Comment:
The .ToString() here is required because you can't implicitly convert System.Uri to string. In OrderWriter there's string concatenation going on, which automatically transforms non-strings into strings, so the .ToString() there was superfluous.